docs/mkdocs/documentation/deploy_kmod.md

```diff
@@ -50,6 +50,48 @@ The reconciliation loop for `Module` runs the following steps:
     2. successful build jobs;
     3. successful signing jobs.
 
+### Soft dependencies between kernel modules
+
+Some setups may require that several kernel modules be loaded to work properly, although the modules do not directly
+depend on each other through symbols.
+`depmod` is usually not aware of those dependencies, and they do not appear in the files it produces.  
+If `mod_a` has a soft dependency on `mod_b`, `modprobe mod_a` will not load `mod_b`.
+
+Soft dependencies can be declared in the `Module` CRD via the
+`.spec.moduleLoader.container.modprobe.modulesLoadingOrder` field:
+
+```yaml
+modulesLoadingOrder:  # optional
+  - mod_a
+  - mod_b
+```
+
+With the configuration above:
+
+- the loading order will be `mod_b`, then `mod_a`;
+- the unloading order will be `mod_a`, then `mod_b`.
```

Comment:
we should probably add that the module to be loaded latest ( the one that depends on all the rest) should be also set in the moduleName field